packages/graph/hash_graph/lib/graph/src/api/rest/link_type.rs

```diff
@@ -35,9 +36,18 @@ use crate::{
 )]
 pub struct LinkTypeResource;
 
+/// Specifies the requirements to a [`Store`] for the link-type REST API.
+///
+/// [`Store`]: crate::store::Store
+pub trait LinkTypeBackend = crud::Read<VersionId, LinkType, Output = Persisted<LinkType>>;
+
 impl RoutedResource for LinkTypeResource {
     /// Create routes for interacting with link types.
-    fn routes<S: StorePool + 'static>() -> Router {
+    fn routes<S>() -> Router
+    where
+        S: StorePool + 'static,
+        for<'pool> S::Store<'pool>: RestApiBackend,
```

Comment:
Will these bounds get simpler in follow-ups? :)

Reply:
No, I wasn't able to reduce them. Please take into account that only one bound was added in this PR, not both :)
(Honestly, I find them pretty simple - esp. compared to the `Tls` bounds 😄. Btw, if we would use `PostgresStore<Tls>` here, the bounds would be unreadable)

Reply:
I was able to reduce it even more 😆  26cd1987500bce2e0439b6bef863b64ea825fd39
```rust
pub trait DataTypeBackend = StorePool + 'static
where
    for<'pool> <Self as StorePool>::Store<'pool>:
        crud::Read<VersionedUri, DataType, Output = DataType>;

// ... for other types as well ...

pub trait RestApiBackend =
    DataTypeBackend + PropertyTypeBackend + LinkTypeBackend + EntityTypeBackend + EntityBackend;

// ...

    fn routes<S: RestApiBackend>() -> Router;
```